docs/AppBar.md

```diff
@@ -0,0 +1,58 @@
+# App Bar
+
+The `<AppBar>` component is an extension of the default AppBar from Material UI that can be resized / collapsed as the page is scrolled. It supports three modes:
+
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 600px" alt="Dropdown Toolbar" src="./images/appBar.png">
+    <img width="100%" style="max-width: 600px" alt="Dropdown Toolbar" src="./images/appBarCollapsed.png">
+</div>
+
+## Usage
+
+```tsx
+import { AppBar } from '@pxblue/react-components';
+...
+// Default: AppBar will resize between collapsedHeight and expandedHeight as the window is scrolled
+<AppBar mode={'dynamic'}>
+  {/* Contents */}
+</AppBar>
+
+// App Bar will stay fixed at the collapsedHeight size
+<AppBar mode={'collapsed'}>
+  {/* Contents */}
+</AppBar>
+
+// App Bar will stay fixed at the expandedHeight size
+<AppBar mode={'expanded'}>
+  {/* Contents */}
+</AppBar>
+```
+
+## API
+
+<div style="overflow: auto;">
+
+| Prop Name         | Description                                      | Type                                         | Required | Default       |
+| ----------------- | ------------------------------------------------ | -------------------------------------------- | -------- | ------------- |
+| animationDuration | Length of the collapse / expand animation (ms)   | `number`                                     | no       | theme default |
+| backgroundImage   | Image to use as the app bar background           | `string`                                     | no       |               |
+| classes           | Style Overrides                                  | `AppBarClasses`                              | no       |               |
+| collapsedHeight   | Height of the AppBar when collapsed              | `number`                                     | no       | theme default |
+| expandedHeight    | Height of the AppBar when expanded               | `number`                                     | no       | 200           |
+| mode              | Behavior mode of the App Bar                     | `'expanded'` \| `'collapsed'` \| `'dynamic'` | no       | 'dynamic'     |
+| scrollThreshold   | Distance to scroll before collapsing the app bar | `number`                                     | no       | 100           |
```

Comment:
`scrollThreshold` default value shall prob be corrected. Same with your comments above the type definition.